src/screens/HomeFlow/Home/components/SocialLinks/components/SocialLinkItem.tsx

```
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: ice License 1.0
+
+import {Touchable} from '@components/Touchable';
+import {COLORS} from '@constants/colors';
+import React, {memo, useCallback} from 'react';
+import {Image, Linking} from 'react-native';
+import {ImageSourcePropType, StyleSheet} from 'react-native';
+import {rem} from 'rn-units';
+
+interface Props {
+  icon: ImageSourcePropType;
+  linkScheme: string | undefined;
+  linkUrl: string;
+}
+
+const openSchemeOrUrl = async ({
+  linkScheme,
+  linkUrl,
+}: {
+  linkScheme?: string;
+  linkUrl: string;
+}) => {
+  try {
+    if (linkScheme) {
+      const supported = await Linking.openURL(linkScheme);
+
+      if (supported) {
+        return Linking.openURL(linkScheme);
+      }
+    }
+  } catch (error) {
+    /**
+     * Ignore. We will try to open linkUrl
+     */
+  }
+
+  return Linking.openURL(linkUrl);
```

better add an empty catch here (or wrap it also with `Linking.canOpenUrl`):
```
Linking.canOpenURL(linkUrl)
              .then(supported => supported && Linking.openURL(linkUrl))
              .catch();
```

Because according to sentry there are some exotic devices that throw exceptions when we're trying to open a link with `Linking.openURL` method,  even if links are totally valid and with `https` scheme.

---

gonna reuse your method